library/computable-reals/computable-reals.lisp

```diff
@@ -33,23 +33,23 @@
 
 (coalton-toplevel
   
- (repr :native cr:creal)
- (define-type Creal)
+  (repr :native (cl:or cr:creal cl:rational))
```

Comment:
Hey It's a nit but can you remove these leading spaces? They appear in a few spots throughout this commit.